packages/elements/src/components/ino-datepicker/ino-datepicker.tsx

```diff
@@ -293,6 +299,8 @@ export class Datepicker implements ComponentInterface {
   private static NUMBERS_WITH_SPECIAL_CHARS = /(\d[^a-z]+)/g
 
   private create() {
+    if (this.disabled) return;
```

Comment:
This function should not be called if the disabled flag is set. We should move this to the `componentDidLoad` function.

Reply:
Done